site/docs/packages/sprinkles.md

```diff
@@ -417,6 +419,40 @@ const properties = defineProperties({
 });
 ```
 
+Properties can also be scoped to container queries.
+
+> 🚧&nbsp;&nbsp;Ensure your target browsers [support container queries]. Vanilla-extract supports the [container query syntax] but does not polyfill the feature in unsupported browsers.
```

Comment:
Looks like you missed the markdown reference for this one

Reply:
Ah, nice catch.